etcdctl/README.md

```diff
@@ -160,9 +160,24 @@ The protobuf encoding of the DeleteRange [RPC response][etcdrpc].
 ```bash
 ./etcdctl put foo bar
 # OK
+./etcdctl put foo1 bar
```

Comment:
@heyitsanthony i have done the changes. Let me know if anything else is required.
Thank you!